resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/internal/proxy/ClientInvoker.java

```diff
@@ -117,26 +120,24 @@ protected ClientInvocation createRequest(Object[] args)
          }
       }
 
-      ClientInvocationBuilder builder = null;
+      ClientConfiguration parentConfiguration=(ClientConfiguration) target.getConfiguration();
+      ClientInvocation clientInvocation = new ClientInvocation(this.webTarget.getResteasyClient(), target.getUri(),
+    		  new ClientRequestHeaders(parentConfiguration), parentConfiguration);
       if (accepts != null)
       {
-         builder = (ClientInvocationBuilder)target.request(accepts);
+         clientInvocation.getHeaders().accept(accepts);
       }
-      else
-      {
-         builder = (ClientInvocationBuilder)target.request();
-      }
-
       for (int i = 0; i < processors.length; i++)
       {
          if (processors != null && processors[i] instanceof InvocationProcessor)
          {
             InvocationProcessor processor = (InvocationProcessor)processors[i];
-            processor.process(builder, args[i]);
+            processor.process(clientInvocation, args[i]);

```

Comment:
Hi Nicolas,

Sorry for the delay. Ok, I get it. It looks like everything is OK. Thanks.

-Ron